zerver/models.py

```diff
@@ -1053,6 +1054,27 @@ def parse_usermessage_flags(val):
         mask <<= 1
     return flags
 
+class Attachment(models.Model):
+    MAX_FILENAME_LENGTH = 100
+    file_name = models.CharField(max_length=MAX_FILENAME_LENGTH, db_index=True)
+    path_id = models.TextField(default='/some/path')
+    owner = models.ForeignKey(UserProfile)
+    message = models.ForeignKey(Message, blank=True, null=True)
+    is_claimed = models.BooleanField(default=False)
```

Comment:
Do we need an `is_claimed` field or can that just be implemented by checking if `message` is null?